jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/cmds/FusekiMain.java

```diff
@@ -414,13 +433,14 @@ protected void processModulesAndArgs() {
 
     private int portNumber(ArgDecl arg) {
         String portStr = getValue(arg);
+        if ( portStr.isEmpty() )
```

Comment:
Not sure if blanks could also be handled here with `StringUtils.isBlank` or .trim()?

Reply:
This is from the command line which will have discarded (unquoted) whitespace.

`--port=""` or `--port=` is clearer whitespace.

There is a argument for applying it to numbers but the command hasn't up to now.